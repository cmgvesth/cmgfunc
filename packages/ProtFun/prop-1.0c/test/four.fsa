>EDA_HUMAN   Q92838 Ectodysplasin A (Ectodermal dysplasia protein) (EDA protein). - Homo sapiens (Human).
MGYPEVERRELLPAAAPRERGSQGCGCGGAPARAGEGNSCLLFLGFFGLSLALHLLTLCC
YLELRSELRRERGAESRLGGSGTPGTSGTLSSLGGLDPDSPITSHLGQPSPKQQPLEPGE
AALHSDSQDGHQMALLNFFFPDEKPYSEEESRRVRRNKRSKSNEGADGPVKNKKKGKKAG
PPGPNGPPGPPGPPGPQGPPGIPGIPGIPGTTVMGPPGPPGPPGPQGPPGLQGPSGAADK
AGTRENQPAVVHLQGQGSAIQVKNDLSGGVLNDWSRITMNPKVFKLHPRSGELEVLVDGT
YFIYSQVEVYYINFTDFASYEVVVDEKPFLQCTRSIETGKTNYNTCYTAGVCLLKARQKI
AVKMVHADISINMSKHTTFFGAIRLGEAPAS
>FGF2_HUMAN  P09038 Heparin-binding growth factor 2 precursor (HBGF-2) (Basic fibroblast growth factor) (BFGF) (Prostatropin). - Homo sapiens (Human).
MAAGSITTLPALPEDGGSGAFPPGHFKDPKRLYCKNGGFFLRIHPDGRVDGVREKSDPHI
KLQLQAEERGVVSIKGVCANRYLAMKEDGRLLASKCVTDECFFFERLESNNYNTYRSRKY
TSWYVALKRTGQYKLGSKTGPGQKAILFLPMSAKS
>GDNF_HUMAN  P39905 Glial cell line-derived neurotrophic factor precursor (Astrocyte- derived trophic factor 1) (ATF-1). - Homo sapiens (Human).
MKLWDVVAVCLVLLHTASAFPLPAGKRPPEAPAEDRSLGRRRAPFALSSDSNMPEDYPDQ
FDDVMDFIQATIKRLKRSPDKQMAVLPRRERNRQAAAANPENSRGKGRRGQRGKNRGCVL
TAIHLNVTDLGLGYETKEELIFRYCSGSCDAAETTYDKILKNLSRNRRLVSDKVGQACCR
PIAFDDDLSFLDDNLVYHILRKHSAKRCGCI
>P53_HUMAN   P04637 Cellular tumor antigen p53 (Tumor suppressor p53) (Phosphoprotein p53) (Antigen NY-CO-13). - Homo sapiens (Human).
MEEPQSDPSVEPPLSQETFSDLWKLLPENNVLSPLPSQAMDDLMLSPDDIEQWFTEDPGP
DEAPRMPEAAPPVAPAPAAPTPAAPAPAPSWPLSSSVPSQKTYQGSYGFRLGFLHSGTAK
SVTCTYSPALNKMFCQLAKTCPVQLWVDSTPPPGTRVRAMAIYKQSQHMTEVVRRCPHHE
RCSDSDGLAPPQHLIRVEGNLRVEYLDDRNTFRHSVVVPYEPPEVGSDCTTIHYNYMCNS
SCMGGMNRRPILTIITLEDSSGNLLGRNSFEVRVCACPGRDRRTEEENLRKKGEPHHELP
PGSTKRALPNNTSSSPQPKKKPLDGEYFTLQIRGRERFEMFRELNEALELKDAQAGKEPG
GSRAHSSHLKSKKGQSTSRHKKLMFKTEGPDSD
