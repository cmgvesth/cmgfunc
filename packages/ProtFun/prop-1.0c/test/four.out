

         ##### ProP v.1.0b ProPeptide Cleavage Site Prediction #####


         ##### Furin-type cleavage site prediction (Arginine/Lysine residues) #####


  391 EDA_HUMAN   
MGYPEVERRELLPAAAPRERGSQGCGCGGAPARAGEGNSCLLFLGFFGLSLALHLLTLCCYLELRSELRRERGAESRLGG      80
SGTPGTSGTLSSLGGLDPDSPITSHLGQPSPKQQPLEPGEAALHSDSQDGHQMALLNFFFPDEKPYSEEESRRVRRNKRS     160
KSNEGADGPVKNKKKGKKAGPPGPNGPPGPPGPPGPQGPPGIPGIPGIPGTTVMGPPGPPGPPGPQGPPGLQGPSGAADK     240
AGTRENQPAVVHLQGQGSAIQVKNDLSGGVLNDWSRITMNPKVFKLHPRSGELEVLVDGTYFIYSQVEVYYINFTDFASY     320
EVVVDEKPFLQCTRSIETGKTNYNTCYTAGVCLLKARQKIAVKMVHADISINMSKHTTFFGAIRLGEAPAS              400
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss..............      80
..............................................................................P.     160
................................................................................     240
................................................................................     320
.......................................................................              400
 
Signal peptide cleavage site predicted:	between pos. 66 and 67: LRS-EL
 
Propeptide cleavage sites predicted:	Arg(R)/Lys(K): 1




Name           Pos    Context     Score  Pred
____________________________v_________________
EDA_HUMAN        8    GYPEVER|RE  0.057    .
EDA_HUMAN        9    YPEVERR|EL  0.128    .
EDA_HUMAN       18    LPAAAPR|ER  0.135    .
EDA_HUMAN       20    AAAPRER|GS  0.102    .
EDA_HUMAN       33    CGGAPAR|AG  0.090    .
EDA_HUMAN       65    CCYLELR|SE  0.089    .
EDA_HUMAN       69    ELRSELR|RE  0.081    .
EDA_HUMAN       70    LRSELRR|ER  0.195    .
EDA_HUMAN       72    SELRRER|GA  0.293    .
EDA_HUMAN       77    ERGAESR|LG  0.112    .
EDA_HUMAN      112    LGQPSPK|QQ  0.064    .
EDA_HUMAN      144    FFFPDEK|PY  0.060    .
EDA_HUMAN      152    YSEEESR|RV  0.096    .
EDA_HUMAN      153    SEEESRR|VR  0.114    .
EDA_HUMAN      155    EESRRVR|RN  0.204    .
EDA_HUMAN      156    ESRRVRR|NK  0.482    .
EDA_HUMAN      158    RRVRRNK|RS  0.182    .
EDA_HUMAN      159    RVRRNKR|SK  0.819 *ProP*
EDA_HUMAN      161    RRNKRSK|SN  0.131    .
EDA_HUMAN      171    GADGPVK|NK  0.060    .
EDA_HUMAN      173    DGPVKNK|KK  0.066    .
EDA_HUMAN      174    GPVKNKK|KG  0.119    .
EDA_HUMAN      175    PVKNKKK|GK  0.156    .
EDA_HUMAN      177    KNKKKGK|KA  0.116    .
EDA_HUMAN      178    NKKKGKK|AG  0.120    .
EDA_HUMAN      240    PSGAADK|AG  0.060    .
EDA_HUMAN      244    ADKAGTR|EN  0.082    .
EDA_HUMAN      263    GSAIQVK|ND  0.066    .
EDA_HUMAN      276    VLNDWSR|IT  0.077    .
EDA_HUMAN      282    RITMNPK|VF  0.057    .
EDA_HUMAN      285    MNPKVFK|LH  0.064    .
EDA_HUMAN      289    VFKLHPR|SG  0.116    .
EDA_HUMAN      327    EVVVDEK|PF  0.060    .
EDA_HUMAN      334    PFLQCTR|SI  0.152    .
EDA_HUMAN      340    RSIETGK|TN  0.061    .
EDA_HUMAN      355    AGVCLLK|AR  0.059    .
EDA_HUMAN      357    VCLLKAR|QK  0.099    .
EDA_HUMAN      359    LLKARQK|IA  0.073    .
EDA_HUMAN      363    RQKIAVK|MV  0.078    .
EDA_HUMAN      375    ISINMSK|HT  0.067    .
EDA_HUMAN      384    TFFGAIR|LG  0.076    .
____________________________^_________________




  155 FGF2_HUMAN  
MAAGSITTLPALPEDGGSGAFPPGHFKDPKRLYCKNGGFFLRIHPDGRVDGVREKSDPHIKLQLQAEERGVVSIKGVCAN      80
RYLAMKEDGRLLASKCVTDECFFFERLESNNYNTYRSRKYTSWYVALKRTGQYKLGSKTGPGQKAILFLPMSAKS          160
................................................................................      80
...........................................................................          160
 
Signal peptide cleavage site predicted:	none
 
Propeptide cleavage sites predicted:	Arg(R)/Lys(K): 0




Name           Pos    Context     Score  Pred
____________________________v_________________
FGF2_HUMAN      27    FPPGHFK|DP  0.059    .
FGF2_HUMAN      30    GHFKDPK|RL  0.070    .
FGF2_HUMAN      31    HFKDPKR|LY  0.214    .
FGF2_HUMAN      35    PKRLYCK|NG  0.069    .
FGF2_HUMAN      42    NGGFFLR|IH  0.101    .
FGF2_HUMAN      48    RIHPDGR|VD  0.101    .
FGF2_HUMAN      53    GRVDGVR|EK  0.093    .
FGF2_HUMAN      55    VDGVREK|SD  0.075    .
FGF2_HUMAN      61    KSDPHIK|LQ  0.066    .
FGF2_HUMAN      69    QLQAEER|GV  0.116    .
FGF2_HUMAN      75    RGVVSIK|GV  0.097    .
FGF2_HUMAN      81    KGVCANR|YL  0.102    .
FGF2_HUMAN      86    NRYLAMK|ED  0.073    .
FGF2_HUMAN      90    AMKEDGR|LL  0.071    .
FGF2_HUMAN      95    GRLLASK|CV  0.088    .
FGF2_HUMAN     106    ECFFFER|LE  0.082    .
FGF2_HUMAN     116    NNYNTYR|SR  0.104    .
FGF2_HUMAN     118    YNTYRSR|KY  0.077    .
FGF2_HUMAN     119    NTYRSRK|YT  0.352    .
FGF2_HUMAN     128    SWYVALK|RT  0.072    .
FGF2_HUMAN     129    WYVALKR|TG  0.134    .
FGF2_HUMAN     134    KRTGQYK|LG  0.080    .
FGF2_HUMAN     138    QYKLGSK|TG  0.052    .
FGF2_HUMAN     144    KTGPGQK|AI  0.059    .
FGF2_HUMAN     154    FLPMSAK|S-  0.089    .
____________________________^_________________




  211 GDNF_HUMAN  
MKLWDVVAVCLVLLHTASAFPLPAGKRPPEAPAEDRSLGRRRAPFALSSDSNMPEDYPDQFDDVMDFIQATIKRLKRSPD      80
KQMAVLPRRERNRQAAAANPENSRGKGRRGQRGKNRGCVLTAIHLNVTDLGLGYETKEELIFRYCSGSCDAAETTYDKIL     160
KNLSRNRRLVSDKVGQACCRPIAFDDDLSFLDDNLVYHILRKHSAKRCGCI                                  240
sssssssssssssssssss.........................................................P...      80
................................................................................     160
.......P...........................................                                  240
 
Signal peptide cleavage site predicted:	between pos. 19 and 20: ASA-FP
 
Propeptide cleavage sites predicted:	Arg(R)/Lys(K): 2




Name           Pos    Context     Score  Pred
____________________________v_________________
GDNF_HUMAN       2    -----MK|LW  0.069    .
GDNF_HUMAN      26    FPLPAGK|RP  0.054    .
GDNF_HUMAN      27    PLPAGKR|PP  0.120    .
GDNF_HUMAN      36    EAPAEDR|SL  0.134    .
GDNF_HUMAN      40    EDRSLGR|RR  0.074    .
GDNF_HUMAN      41    DRSLGRR|RA  0.163    .
GDNF_HUMAN      42    RSLGRRR|AP  0.124    .
GDNF_HUMAN      73    FIQATIK|RL  0.059    .
GDNF_HUMAN      74    IQATIKR|LK  0.140    .
GDNF_HUMAN      76    ATIKRLK|RS  0.062    .
GDNF_HUMAN      77    TIKRLKR|SP  0.802 *ProP*
GDNF_HUMAN      81    LKRSPDK|QM  0.062    .
GDNF_HUMAN      88    QMAVLPR|RE  0.090    .
GDNF_HUMAN      89    MAVLPRR|ER  0.108    .
GDNF_HUMAN      91    VLPRRER|NR  0.146    .
GDNF_HUMAN      93    PRRERNR|QA  0.152    .
GDNF_HUMAN     104    ANPENSR|GK  0.088    .
GDNF_HUMAN     106    PENSRGK|GR  0.071    .
GDNF_HUMAN     108    NSRGKGR|RG  0.099    .
GDNF_HUMAN     109    SRGKGRR|GQ  0.254    .
GDNF_HUMAN     112    KGRRGQR|GK  0.251    .
GDNF_HUMAN     114    RRGQRGK|NR  0.091    .
GDNF_HUMAN     116    GQRGKNR|GC  0.095    .
GDNF_HUMAN     137    GLGYETK|EE  0.058    .
GDNF_HUMAN     143    KEELIFR|YC  0.112    .
GDNF_HUMAN     158    AETTYDK|IL  0.054    .
GDNF_HUMAN     161    TYDKILK|NL  0.067    .
GDNF_HUMAN     165    ILKNLSR|NR  0.072    .
GDNF_HUMAN     167    KNLSRNR|RL  0.070    .
GDNF_HUMAN     168    NLSRNRR|LV  0.767 *ProP*
GDNF_HUMAN     173    RRLVSDK|VG  0.064    .
GDNF_HUMAN     180    VGQACCR|PI  0.080    .
GDNF_HUMAN     201    LVYHILR|KH  0.093    .
GDNF_HUMAN     202    VYHILRK|HS  0.074    .
GDNF_HUMAN     206    LRKHSAK|RC  0.095    .
GDNF_HUMAN     207    RKHSAKR|CG  0.222    .
____________________________^_________________




  393 P53_HUMAN   
MEEPQSDPSVEPPLSQETFSDLWKLLPENNVLSPLPSQAMDDLMLSPDDIEQWFTEDPGPDEAPRMPEAAPPVAPAPAAP      80
TPAAPAPAPSWPLSSSVPSQKTYQGSYGFRLGFLHSGTAKSVTCTYSPALNKMFCQLAKTCPVQLWVDSTPPPGTRVRAM     160
AIYKQSQHMTEVVRRCPHHERCSDSDGLAPPQHLIRVEGNLRVEYLDDRNTFRHSVVVPYEPPEVGSDCTTIHYNYMCNS     240
SCMGGMNRRPILTIITLEDSSGNLLGRNSFEVRVCACPGRDRRTEEENLRKKGEPHHELPPGSTKRALPNNTSSSPQPKK     320
KPLDGEYFTLQIRGRERFEMFRELNEALELKDAQAGKEPGGSRAHSSHLKSKKGQSTSRHKKLMFKTEGPDSD            400
................................................................................      80
................................................................................     160
................................................................................     240
................................................................................     320
.........................................................................            400
 
Signal peptide cleavage site predicted:	none
 
Propeptide cleavage sites predicted:	Arg(R)/Lys(K): 0




Name           Pos    Context     Score  Pred
____________________________v_________________
P53_HUMAN       24    TFSDLWK|LL  0.062    .
P53_HUMAN       65    GPDEAPR|MP  0.114    .
P53_HUMAN      101    SSVPSQK|TY  0.066    .
P53_HUMAN      110    QGSYGFR|LG  0.080    .
P53_HUMAN      120    LHSGTAK|SV  0.134    .
P53_HUMAN      132    YSPALNK|MF  0.064    .
P53_HUMAN      139    MFCQLAK|TC  0.064    .
P53_HUMAN      156    TPPPGTR|VR  0.071    .
P53_HUMAN      158    PPGTRVR|AM  0.094    .
P53_HUMAN      164    RAMAIYK|QS  0.070    .
P53_HUMAN      174    HMTEVVR|RC  0.080    .
P53_HUMAN      175    MTEVVRR|CP  0.178    .
P53_HUMAN      181    RCPHHER|CS  0.095    .
P53_HUMAN      196    PPQHLIR|VE  0.088    .
P53_HUMAN      202    RVEGNLR|VE  0.060    .
P53_HUMAN      209    VEYLDDR|NT  0.071    .
P53_HUMAN      213    DDRNTFR|HS  0.090    .
P53_HUMAN      248    CMGGMNR|RP  0.068    .
P53_HUMAN      249    MGGMNRR|PI  0.147    .
P53_HUMAN      267    SGNLLGR|NS  0.106    .
P53_HUMAN      273    RNSFEVR|VC  0.088    .
P53_HUMAN      280    VCACPGR|DR  0.121    .
P53_HUMAN      282    ACPGRDR|RT  0.074    .
P53_HUMAN      283    CPGRDRR|TE  0.493    .
P53_HUMAN      290    TEEENLR|KK  0.082    .
P53_HUMAN      291    EEENLRK|KG  0.072    .
P53_HUMAN      292    EENLRKK|GE  0.091    .
P53_HUMAN      305    LPPGSTK|RA  0.065    .
P53_HUMAN      306    PPGSTKR|AL  0.214    .
P53_HUMAN      319    SSSPQPK|KK  0.073    .
P53_HUMAN      320    SSPQPKK|KP  0.110    .
P53_HUMAN      321    SPQPKKK|PL  0.233    .
P53_HUMAN      333    YFTLQIR|GR  0.075    .
P53_HUMAN      335    TLQIRGR|ER  0.105    .
P53_HUMAN      337    QIRGRER|FE  0.089    .
P53_HUMAN      342    ERFEMFR|EL  0.095    .
P53_HUMAN      351    NEALELK|DA  0.085    .
P53_HUMAN      357    KDAQAGK|EP  0.080    .
P53_HUMAN      363    KEPGGSR|AH  0.091    .
P53_HUMAN      370    AHSSHLK|SK  0.091    .
P53_HUMAN      372    SSHLKSK|KG  0.066    .
P53_HUMAN      373    SHLKSKK|GQ  0.157    .
P53_HUMAN      379    KGQSTSR|HK  0.092    .
P53_HUMAN      381    QSTSRHK|KL  0.062    .
P53_HUMAN      382    STSRHKK|LM  0.463    .
P53_HUMAN      386    HKKLMFK|TE  0.066    .
____________________________^_________________




